src/main/java/htsjdk/samtools/SAMFileWriterFactory.java

```diff
@@ -467,6 +472,15 @@ private CRAMFileWriter createCRAMWriterWithSettings(
             final boolean presorted,
             final File outputFile,
             final File referenceFasta) {
+
+        final CRAMReferenceSource referenceSource ;
+        if (referenceFasta == null) {
+            log.warn("Reference fasta is not provided when writing CRAM file " + outputFile.getAbsolutePath());
+            referenceSource = ReferenceSource.getDefaultCRAMReferenceSource();
```

Comment:
Looks not to be covered by a test, would it be easy to add one?

Reply:
I added a test